setupbase.py

```diff
@@ -135,7 +135,6 @@ def find_package_data():
         pjoin(components, "bootstrap", "js", "bootstrap.min.js"),
         pjoin(components, "bootstrap-tour", "build", "css", "bootstrap-tour.min.css"),
         pjoin(components, "bootstrap-tour", "build", "js", "bootstrap-tour.min.js"),
-        pjoin(components, "es6-promise", "*.js"),
```

Comment:
I don't know exactly where npm puts these files, but I suspect they should still be included in the tarball somehow...